akka-http-circe/src/main/scala/de/heikoseeberger/akkahttpcirce/CirceSupport.scala

```diff
@@ -67,9 +72,7 @@ trait CirceSupport {
     */
   implicit def circeToEntityMarshaller[A](
       implicit encoder: Encoder[A],
-      printer: Json => String = Printer.noSpaces.pretty
+      printer: Json ⇒ String = Printer.noSpaces.pretty
   ): ToEntityMarshaller[A] =
-    Marshaller.StringMarshaller
-      .wrap(`application/json`)(printer)
-      .compose(encoder.apply)
+    jsonStringMarshaller.compose(printer).compose(encoder.apply)
```

Comment:
What's the exact difference btw. the new and old implementation? Please notice that the old implementation follows [SprayJsonSupport](https://github.com/akka/akka/blob/master/akka-http-marshallers-scala/akka-http-spray-json/src/main/scala/akka/http/scaladsl/marshallers/sprayjson/SprayJsonSupport.scala).